### PR TITLE
FlagEval-debate blog - publication Thursday

### PR DESCRIPTION
**Pull request by xuanricheng**
Congratulations! You've made it this far! Once merged, the article will appear at https://huggingface.co/blog. Official articles 
require additional reviews. Alternatively, you can write a community article following the process [here](https://huggingface.co/blog-explorers).

## Preparing the Article

You're not quite done yet, though. Please make sure to follow this process (as documented [here](https://github.com/huggingface/blog/tree/main?tab=readme-ov-file#how-to-write-an-article-)):

- [x] Add an entry to [_blog.yml](https://github.com/huggingface/blog/blob/main/_blog.yml).
- [x] Add a thumbnail. There are no requirements here, but there is a [template](https://github.com/huggingface/blog?tab=readme-ov-file#how-to-get-a-nice-responsive-thumbnail) if it's helpful.
- [x] Check you use a short title and blog path.
- [x] Upload any additional assets (such as images) to the Documentation Images [repo](https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog). This is to reduce bloat in the GitHub base repo when cloning and pulling. Try to have small images to avoid a slow or expensive user experience.
- [ ] Add metadata (such as authors) to your `md` file. You can also specify `guest` or `org` for the authors.
- [ ] Ensure the publication date is correct.
- [ ] Preview the content. A quick way is to paste the markdown content in https://huggingface.co/new-blog. **Do not click publish**, this is just a way to do an early check.

Here is an example of a complete PR: https://github.com/huggingface/blog/pull/2382

## Getting a Review

Please make sure to get a review from someone on your team or a co-author.
Once this is done and once all the steps above are completed, you should be able to merge.
There is no need for additional reviews if you and your co-authors are happy and meet all of the above.

Feel free to add @pcuenca as a reviewer if you want a final check. Keep in mind he'll be biased toward light reviews
(e.g., check for proper metadata) rather than content reviews unless explicitly asked.

### CODE REVIEW

**Comment by xuanricheng** [comment]
Hey, @xianbaoqian @clefourrier I just submitted refined version of flageval-debae blog. please help to review. Thanks!

File: _blog.yml [hunk 1]
```diff
@@ -4976,4 +4976,10 @@
     - community
     - research
     - datasets
-    - guide
\ No newline at end of file
+    - guide
+
+- local: debate
+  title: "Letting Large Models Debate: The First Multilingual LLM Debate Competition"
+  author: xuanricheng
```
Comment (clefourrier) [inline _blog.yml:4983]: You need to add guest: true

File: debate.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  org: BAAI
+- user: Lyfly2024
+  guest: true
+  org: BAAI
+- user: xianbaoqian
+  guest: true
+  org: Huggingface
+- user: clefourrier
+  guest: true
+  org: Huggingface
```
Comment (clefourrier) [inline debate.md:39]: Here you can remove guest :)

**Review by clefourrier — APPROVED** [review]

File: _blog.yml [hunk 3]
```diff
@@ -4976,4 +4976,10 @@
...
+  thumbnail: blog/assets/leaderboards-on-the-hub/thumbnail_flageval.png
+  date: November 20, 2024
```
Comment (clefourrier) [inline _blog.yml:4985]: We already have a blog planned for tomorrow, can we publish yours on Thursday?

File: debate.md (hunk 2 above)
Comment (xuanricheng) [reply debate.md]: Thank you! remove is complete.

File: _blog.yml (hunk 3 above)
Comment (xuanricheng) [reply _blog.yml:4986]: Thursday works perfectly, thank you!

**Review by xianbaoqian — APPROVED** [review]

**Pull request merged**